TrafficSimulator/Assets/UI/OverlayController.cs

```diff
@@ -84,7 +84,8 @@ void Awake()
             _fpvButton.clicked += FPVButtonOnClicked;
             GreyOutButton(_fpvButton);
 
-            _statisticsButton = _doc.rootVisualElement.Q<Button>("Statistics");
+            
+            /*_statisticsButton = _doc.rootVisualElement.Q<Button>("Statistics");
```

Comment:
Commented code should be removed

Reply:
Think @Marre-B wrote this class, so uncomfortable with removing everything that might be needed to be removed. Will fix later.